nwbwidgets/analysis/placefields.py

```diff
@@ -80,7 +80,11 @@ def compute_2d_occupancy(pos, pos_tt, edges_x, edges_y, speed_thresh=0.03):
     """
 
     sampling_period = (np.max(pos_tt) - np.min(pos_tt)) / len(pos_tt)
-    is_running = compute_speed(pos, pos_tt) > speed_thresh
+    if len(velocity) == 0:
+        is_running = compute_speed(pos, pos_tt) > speed_thresh
+    else:
+        is_running = self.velocity > speed_thresh
```

Comment:
you should compute speed from velocity (https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html)